src/DocumentDB.ChangeFeedProcessor.IntegrationTests/V3CompatibilityTests.cs

```diff
@@ -198,6 +212,39 @@ public async Task Schema_OnV2MigrationMaintainLeaseToken()
             }
         }
 
+        private async Task WaitUntilLeaseStoreIsInitializedAsync()
+        {
+            bool infoExists = false;
+            bool lockExists = false;
+            using (DocumentClient client = new DocumentClient(this.LeaseCollectionInfo.Uri, this.LeaseCollectionInfo.MasterKey, this.LeaseCollectionInfo.ConnectionPolicy))
+            {
+                Uri collectionUri = UriFactory.CreateDocumentCollectionUri(this.LeaseCollectionInfo.DatabaseName, this.LeaseCollectionInfo.CollectionName);
+
+                IDocumentQuery<JObject> query = client.CreateDocumentQuery<JObject>(collectionUri, "SELECT * FROM c").AsDocumentQuery();
```

Comment:
this should be SELECT * FROM c WHERE CONTAINS(c.id, ".info") OR CONTAINS(c.id, ".lock")

Reply:
Done